src/adagrams.js

```diff
@@ -1,8 +1,84 @@
+// Distribution of Letters
+const letterPool = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D','D','D','D',
+                    'E','E','E','E','E','E','E','E','E','E','E','E',
+                    'F','F','G','G','G','H','H','I','I','I','I','I','I','I','I','I',
+                    'J','K','L','L','L','L','M','M','N','N','N','N','N','N',
+                    'O','O','O','O','O','O','O','O','P','P','Q','R','S','S','S','S',
+                    'T','T','T','T','T','T','U','U','U','U','V','V','W','W',
+                    'X','Y','Y','Z']
+
+const scoredWords = {};
+scoredWords['A'] = 1;
+scoredWords['B'] = 3;
+scoredWords['C'] = 3;
+scoredWords['D'] = 2;
+scoredWords['E'] = 1;
+scoredWords['F'] = 4;
+scoredWords['G'] = 2;
+scoredWords['H'] = 4;
+scoredWords['I'] = 1;
+scoredWords['J'] = 8;
+scoredWords['K'] = 5;
+scoredWords['L'] = 1;
+scoredWords['M'] = 3;
+scoredWords['N'] = 1;
+scoredWords['O'] = 1;
+scoredWords['P'] = 3;
+scoredWords['Q'] = 10;
+scoredWords['R'] = 1;
+scoredWords['S'] = 1;
+scoredWords['T'] = 1;
+scoredWords['U'] = 1;
+scoredWords['V'] = 4;
+scoredWords['W'] = 4;
+scoredWords['X'] = 8;
+scoredWords['Y'] = 4;
+scoredWords['Z'] = 10;
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const shuffled = letterPool.sort(()=> 0.5 - Math.random());
+    let selected = shuffled.slice(0, 10);
+    return selected;
+  },
+  usesAvailableLetters(input, lettersInHand) {
+    input = Array.from(input);
```

Comment:
you are re-assigning the `input` array with the value `Array.from(input)`, which feels kind of weird. It may be better to find a different variable, like `const letters = Array.from(input);`